dkist/net/attrs.py

```diff
@@ -157,6 +159,8 @@ class Embargoed(_SimpleAttr):
         A boolean determining if a dataset currently under embargo.
     """
     def __init__(self, is_embargoed: bool):
+        if isinstance(is_embargoed, str) and is_embargoed.lower() == "false":
+            is_embargoed = False
```

Comment:
What about `"true"` :wink: 

More over I think that if it's not `bool(True)` then it should be False. i.e. why is `a.Embargoed("wibble")` the same as True?